os2forms.info.yml

```diff
@@ -6,27 +6,28 @@ core: 8.x
 core_version_requirement: ^8 || ^9
 
 dependencies:
+  - 'drupal:advancedqueue'
```

Comment:
there is no dependency on **advancedqueue** in os2forms in general module.

Currently it potentially possible to have os2forms installation without os2forms_forloeb module enabled, which means requiring **advancedqueue** is excessive.

It would be better to force the dependencies via info if and when they are really required.

Reply:
You're absolutely right. I've fixed this.